android/MapboxGLAndroidSDKTestApp/src/main/res/layout/activity_infowindow_concurrent.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="@color/primary"
+        android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar" />
+
+    <com.mapbox.mapboxsdk.views.MapView
+        android:id="@+id/mapView"
```

Comment:
@zugaldia Let's change the id to something like `@+id/infoWindowConcurrentMapView`.  While there shouldn't be any runtime issues due to namespacing, I unfortunately got us into the habit of using `@+id/mapView` in all the TestApp examples which makes immediate sight recognition more difficult than it should be.